contrib/datawave-quickstart/bin/query.sh

```diff
@@ -131,21 +131,32 @@ function setQueryIdFromResponse() {
 }
 
 function prettyPrintJson() {
-    local PY=$( which python )
-    if [ -n "${PY}" ] ; then
-        echo "${1}" | ${PY} -c 'from __future__ import print_function;import sys,json;data=json.loads(sys.stdin.read()); print(json.dumps(data, indent=2, sort_keys=True))'
-        local exitStatus=$?
-        echo
-        if [ "${exitStatus}" != "0" ] ; then
-           printRawResponse "${1}"
-           warn "Python encountered error. Printed response without formatting"
-           echo
+    PY_CMD='"from __future__ import print_function; import sys,json; data=json.loads(sys.stdin.read()); print(json.dumps(data, indent=2, sort_keys=True)"'
+        PY3=$(which python3 2>/dev/null)
+        PY2=$(which python2 2>/dev/null)
+        if [ -n "${PY3}" ] ; then
+                echo "${1}" | "${PY3}" -c "${PY_CMD}"
+            local exitStatus=$?
```

Comment:
There are some places throughout your changes where lines are indented incorrectly. Here in this example, you can see the extra indentation in front of the `PY3` and `PY2` variables all the way down to the closing of the if statement. We should probably change that, and make sure each line is properly indented.